src/transform/DimImagesTransform.js

```diff
@@ -1,27 +1,40 @@
 import './DimImagesTransform.css'
 const CLASS = 'pagelib_dim_images'
 
-// todo: only require a Document
 /**
- * @param {!Window} window
+ * @param {!Document} document
  * @param {!boolean} enable
  * @return {void}
  */
-const dim = (window, enable) => {
-  window.document.querySelector('html')
-    .classList[enable ? 'add' : 'remove'](CLASS)
+const dimImages = (document, enable) => {
+  document.querySelector('html').classList[enable ? 'add' : 'remove'](CLASS)
 }
 
-// todo: only require a Document
 /**
+ * @deprecated Use dimImages instead, which only requires a Document
```

Comment:
Is there any need to keep this around?

Reply:
Yes, the apps still use this. We could get rid of them once they use the newer functions.